tst/com/amazon/corretto/crypto/provider/test/SecurityPropertyTester.java

```diff
@@ -31,12 +31,22 @@ public static void main(String[] args) throws Exception {
 
     String infoStr = NATIVE_PROVIDER.getInfo();
     System.out.println("Security Provider : " + infoStr);
-    assertTrue(infoStr.startsWith(NATIVE_PROVIDER.getName()));
+    String[] tokens = infoStr.split("[()]");
+    String[] semver = tokens[0].split("[\\s+-]");
+    String[] addInfo = tokens[1].split(",");
+    assertEquals(NATIVE_PROVIDER.getName(), semver[0]);
+    assertEquals(NATIVE_PROVIDER.getVersionStr(), semver[1]);
     if (NATIVE_PROVIDER.isFips()) {
-      assertTrue(infoStr.endsWith(NATIVE_PROVIDER.getVersionStr() + "-FIPS"));
+      assertEquals("FIPS", semver[2]);
     } else {
-      assertTrue(infoStr.endsWith(NATIVE_PROVIDER.getVersionStr()));
+      assertFalse(infoStr.contains("FIPS"));
     }
+    if (NATIVE_PROVIDER.isExperimentalFips()) {
+      assertEquals("EXP", semver[3]);
+    } else {
+      assertFalse(infoStr.contains("EXP"));
+    }
+    assertEquals(NATIVE_PROVIDER.getAwsLcVersionStr(), addInfo[0]);
```

Comment:
This will break some of our other builds which have not been updated to account for the new cmake variable.